projects/plugins/crm/includes/ZeroBSCRM.FormatHelpers.php

```diff
@@ -799,7 +799,23 @@ function zeroBSCRM_html_companyTimeline($companyID=-1,$logs=false,$companyObj=fa
                          	if (isset($log['type'])) echo esc_html( $log['type'] ); 
                          }
                          ?></h3>
-                        <p><?php if (isset($log['shortdesc'])) echo esc_html( $log['shortdesc'] ); ?><?php if (isset($log['author'])) echo " &mdash; " . esc_html( $log['author'] ) ?><?php if (isset($log['nicetime'])) echo ' &mdash; <i class="clock icon"></i>' . esc_html( $log['nicetime'] ) ?></p>
+						<p>
+						<?php
+						if ( isset( $log['shortdesc'] ) ) {
+							echo wp_kses( $log['shortdesc'], array( 'i' => array( 'class' => true ) ) );
+							?>
+							<?php
+							if ( isset( $log['author'] ) ) {
+								echo ' &mdash; ' . esc_html( $log['author'] );
+							}
+							?>
+							<?php
```

Comment:
and here the same

Reply:
Oh good point, I had the same thought as I was reformatting the existing code but then forgot about removing it and testing if it all still worked fine. Which it does.... added a new commit now.